zh_cn.lang

```diff
@@ -5,27 +5,27 @@ gui.xaero_miscellaneous=杂项
 gui.xaero_miscellaneous_settings=杂项设置
 gui.xaero_notification_settings=通知设置
 gui.xaero_better_pvp_settings=Better PvP 设置
-gui.xaero_box_potion_effects=一个显示当前激活的药水效果的界面。
-gui.xaero_box_armour_status=一个显示当前装备和主手物品信息的界面。
-gui.xaero_box_sprint=使用一个单独的按键（可配置的）来疾跑。
-gui.xaero_box_sneak=允许你切换潜行状态，并且在潜行状态中不需要按住任何按键。
-gui.xaero_box_notifications=为一些信息启用屏幕通知，如低生命值、低饱食度、低氧气量、爆炸危险以及被射箭的通知。
-gui.xaero_box_xp=获得或者失去经验值时将会有提示。
-gui.xaero_box_quick_use=一个类似于动作条的热键系统，可以让你在不将主手物品换掉的情况下使用指定的物品。按住热键来持续使用物品，就像按住右键时一样。比如：放置火把、TNT、扔药水、吃东西、喝东西等等。
-gui.xaero_box_minimap=一个显示你周围环境与实体的小地图。
+gui.xaero_box_potion_effects=显示当前激活的药水效果的界面。
+gui.xaero_box_armour_status=显示你当前穿的盔甲和持有物品信息的界面。
+gui.xaero_box_sprint=通过按下一个绑定的键（可配置）来疾跑。
```

Comment:
The old translation seems more correct to me when translated back. The main point of this feature is that you only use 1 key to sprint and don't have to also hold the W key to move forward.

Reply:
I edited the translation to emphasize that sprinting only requires one key press :)